<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/vslbvascvm/Pictures/Saved Pictures/Screenshot 2024-02-29 195650.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="condition">
       <property name="text">
        <string>Формула для расчета:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="formula">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/vslbvascvm/Pictures/Saved Pictures/Screenshot 2024-02-29 201130.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Введите число m:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_count">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Введите число n:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="n_count">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Результат:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="result"/>
     </item>
     <item>
      <widget class="QPushButton" name="calculate_button">
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
